backend/subroker/views.py

```diff
@@ -336,7 +361,56 @@ def content_detail(request, content_id):
         return HttpResponseNotAllowed(['GET'])
 
 def content_recommendation(request, user_id):
-    return HttpResponse("recommendation")
+    if request.method == "GET":
+        if not request.user.is_authenticated:
+            #ERR 401 : Not Authenticated
+            return HttpResponse(status=401)
+
+        try:
+            user = User.objects.get(id=user_id)
```

Comment:
why do we need user_id here? what about just using request.user.id?

Reply:
@cw10108 I think we don't need user_id in this project, but for expansion, it is better to separate.